recipes/epysurv/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "epysurv" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  url: https://github.com/jarnorfb/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 56383ce024e2ad76c1ac300065efe555cc57fcbc8d7ab00052b310195470e80c
+
+build:
+  # noarch: python
+  number: 0
+  merge_build_host: True  # [win]
```

Comment:
You are skipping windows so you don't need this line.

Reply:
Actually we don't want to skip windows. I believe many epidemiologist needs to use windows in their organizations, so I removed this.

Reply:
If your package does not contain source code that needs compiled, please add `noarch: python` to the `build:` section